Assignment5-click/calculator.py

```diff
@@ -0,0 +1,48 @@
+import click
+
+@click.group()
+def cli():
+    pass
+
+@cli.command()
+@click.argument('firstnum')
+@click.argument('secondnum')
+def sum(firstnum,secondnum):
+    '''
+       Return sum of two nos
+    '''
+    click.echo(int(firstnum)+int(secondnum))
```

Comment:
Main thing I would do in this PR is add in some error catching.  If I managed to invoke the sum function with "one" and "two" as my numbers, it will error out instead of gracefully failing with a "Sorry, please use integers" type message.